docs/site/src/pages/style/color/color.js

```diff
@@ -0,0 +1,174 @@
+  const colorGroups = [];
+  const neutralGroups = [];
+
+  function getColorGroup(styles, colorName, showAltPalette) {
+    const mainPalette = [50, 100, 200, 300, 400, 500, 600, 700, 800, 900];
+    const altPalette = ['A100', 'A200', 'A400', 'A700'];
+    const cssColor = colorName.replace(' ', '').replace(colorName.charAt(0),
+     colorName.charAt(0).toLowerCase());
+    const colorGroupStyle = classes.colorGroup;
+    const colorsList = [];
+    mainPalette.forEach((mainValue) => {
+      colorsList.push(getColorBlock(styles, cssColor, mainValue));
+    }, this);
+
+    if (showAltPalette) {
+      altPalette.forEach((altValue) => {
+        colorsList.push(getColorBlock(styles, cssColor, altValue));
+      }, this);
+    }
+
+    return (
+      <ul className={colorGroupStyle}>
+        {React.Children.toArray(getColorBlock(styles, cssColor, 500, color))}
+        <div className={classes.blockSpace} />
+        {React.Children.toArray(colorsList)}
+      </ul>
+    );
+  }
+
+  function getColorBlock(styles, colorName, colorValue, colorTitle) {
+    const bgColorText = colorName + colorValue;
+    const bgColor = colors[colorName][colorValue];
+    let fgColor = colors.fullBlack;
+    const contrastRatio = getContrastRatio(bgColor, fgColor);
+    let blockTitle;
+    let spaceRequired = false;
+    const colorValueString = colorValue.toString();
+
+    if ((colorValueString.indexOf('A1')) === 0) {
+      spaceRequired = true;
+    }
+
+    if (contrastRatio < 7) fgColor = color.fullWhite;
+    if (colorTitle) {
+      blockTitle = (
+        <span className={styles.name}>
+          {colorTitle}
+        </span>
+      );
+    }
+
+    const rowStyle = {
+      backgroundColor: bgColor,
+      color: fgColor,
+      listStyle: 'none',
+      padding: 15,
+    };
+    const rowStyleSpace = {
+      backgroundColor: bgColor,
+      color: fgColor,
+      listStyle: 'none',
+      padding: 15,
+      borderTop: '4px #eeeeee solid',
+    };
+
+    if (spaceRequired === true) {
+      return (
+        <li style={rowStyleSpace}>
+          {blockTitle}
+          {getColorName(styles, bgColorText, bgColor)}
+        </li>
+      );
+    }
+    return (
+      <li style={rowStyle}>
+        {blockTitle}
+        {getColorName(styles, bgColorText, bgColor)}
+      </li>
+    );
+  }
+
+  function getColorName(styles, text, colorValue) {
+    return (
+      <div className={styles.colorContainer}>
+        <span>{text}</span>
+        <span className={styles.colorValue}>{colorValue.toUpperCase()}</span>
+      </div>
+    );
+  }
+
+
+  mainColors.forEach((maincolor) => {
+    colorGroups.push(getColorGroup(classes, maincolor, true));
+  }, this);
+
+
+  neutralColors.forEach((neutralcolor) => {
+    neutralGroups.push(getColorGroup(classes, neutralcolor, false));
+  }, this);
+
+  return (
+    <div className={classes.divSection} >
+      <div>
+        {React.Children.toArray(colorGroups)}
```

Comment:
Why using `React.Children.toArray`? That's terrible for performances. I don't think you need them.